src/scripts/cancel_older_approvals.sh

```diff
@@ -2,7 +2,7 @@
 set -eo pipefail
 
 BASE_URL="https://circleci.com/api/v2"
-TOKEN="circle-token=${CIRCLECI_API_TOKEN}"
+TOKEN="circle-token=${CIRCLECI_TOKEN}"
```

Comment:
This is instantiated earlier as `CIRCLECI_TOKEN` over at
https://github.com/coda/circleci-utils/blob/main/src/commands/cancel-older-awaiting-approvals.yml#L20

<img width="804" alt="Screen Shot 2022-02-11 at 2 40 22 PM" src="https://user-images.githubusercontent.com/92751744/153680428-02319082-df77-4329-8691-51c2b3c34bf0.png">